Nrfx 7252 rename SDP to HPF (#21515)
nrfconnect/sdk-nrf: applications/hpf/gpio/README.rst

.. _hpf_gpio:

High-Performance Framework GPIO
###############################

> This application demonstrates how to create a software peripheral on FLPR in High-Performance Framework (HPF) by implementing a subset of Zephyr GPIO API.

Reviewer: ```suggestion
This application demonstrates how to create a software peripheral on FLPR in High-Performance Framework (HPF) by implementing a subset of Zephyr's GPIO API.
```

> This application demonstrates how to create High-Performance Framework (HPF) on FLPR in Zephyr threadless mode by implementing a subset of Zephyr GPIO API.

Reviewer 2: ```suggestion
This application demonstrates how to create a software peripheral on FLPR in High-Performance Framework (HPF) by implementing a subset of Zephyr GPIO API.
```

.. _hpf_gpio:

> High-Performance Framework GPIO
> ########

Reviewer: ```suggestion
High-Performance Framework GPIO
###############################
```

> This application demonstrates how to create High Performance Framework on FLPR in Zephyr threadless
> mode by implementing a subset of Zephyr GPIO API.

Reviewer: ```suggestion
This application demonstrates how to create a high-performance framework on FLPR in Zephyr's threadless mode by implementing a subset of the Zephyr GPIO API.
```

Reviewer: need to remove sdp_gpio refs from the following files: 
- release-notes-2.9.0.rst
- release-notes-2.9.0-nRF54H20-1.rst
this should fix the build

> This application demonstrates how to create High-Performance Framework (HPF) on FLPR in Zephyr
> threadless mode by implementing a subset of Zephyr GPIO API.

Reviewer: ```suggestion
This application demonstrates how to create High-Performance Framework (HPF) on FLPR in Zephyr's threadless mode by implementing a subset of Zephyr GPIO API.
```

> .. _sdp_gpio:

Reviewer 3: I think it is best to retain this label.

Author: Could you explain why? High-Performance Framework (HPF) is the new official name and Software Defined Peripheral (SDP) should not be used anymore.

Reviewer 3: It is a label and not visible in docs. If we retain it, the advantage is that we need not edit old release notes. Usually when we rename a page, the older labels are retained.

Author: Ok, changed back

Reviewer 3: What I meant is we could have both the labels, retain the old and add the new one for future use. :)

Author: Added new label and restored the old one.
